src/TriggerBinding/SqlTriggerUtils.cs

```diff
@@ -126,5 +126,35 @@ internal static string GetBracketedLeasesTableName(string userDefinedLeasesTable
             return string.IsNullOrEmpty(userDefinedLeasesTableName) ? string.Format(CultureInfo.InvariantCulture, LeasesTableNameFormat, $"{userFunctionId}_{userTableId}") :
                 string.Format(CultureInfo.InvariantCulture, UserDefinedLeasesTableNameFormat, $"{userDefinedLeasesTableName.AsBracketQuotedString()}");
         }
+
+        /// <summary>
+        /// Returns the number of changes in the change tracking table of ID of the user table.
+        /// </summary>
+        /// <param name="connectionString">SQL connection string used to connect to user database</param>
+        /// <param name="userTable">SqlObject user table</param>
+        /// <param name="logger">Facilitates logging of messages</param>
+        /// <param name="cancellationToken">Cancellation token to pass to the command</param>
+        /// <exception cref="InvalidOperationException">Thrown in case of error when querying the object ID for the user table</exception>
+        internal static async Task<int> GetChangeCountFromChangeTrackingAsync(string connectionString, SqlObject userTable, ILogger logger, CancellationToken cancellationToken)
+        {
+            string getChangeCountCommand = $"SELECT COUNT_BIG(*) FROM CHANGETABLE(CHANGES {userTable.FullName}, null) AS ChTbl;";
```

Comment:
```suggestion
            string getChangeCountCommand = $"SELECT COUNT_BIG(*) FROM CHANGETABLE(CHANGES {userTable.BracketQuotedFullName}, null) AS ChTbl;";
```

Reply:
how will the BracketQuotedFullName look? Like ex: `[dbo].[Products]`

Reply:
We need to bracket (or single quote depending on where it's used) ALL user object names, since otherwise we leave this open to a SQL injection attack (and it just plain won't work for tables with spaces in them) otherwise

And yes - it'll bracket quote the schema + table name. https://github.com/Azure/azure-functions-sql-extension/blob/35ad03153a1e557d3a61f4a938b551ca4f083fb7/src/SqlObject.cs#L77C131-L77C152